layouts/partials/team.html

```diff
@@ -18,7 +18,11 @@ <h3 class="py-2 text-2xl md:text-4xl font-extrabold">{{ with .title }} {{ . }} {
                     {{"<!-- team member -->" | safeHTML}}
                     <a href="{{.bioLink}}">
                         <li class="mt-6 sm:mt-0 bg-[#ECF6FF] rounded-xl h-full">
-                            <img src="{{ .image | absURL }}" alt="{{ .name }}" class="rounded-t-xl covatar">
+                            {{ $img := resources.Get .image }}
+                            {{ $img := $img.Resize "x430" }}
+                            {{ $webp := $img.Process "resize x430 webp q80"}}
```

Comment:
After noticing how large some of the team images are while testing the network with a slower connection, I thought it would be a good idea to convert and resize these as well.

Reply:
Did it improve latency?

Reply:
It did. I set my browser to use a 3G network, before making changes, and it took a very long time for all of our images to load. For example, we were loading a `1.1 MB` sized image for my photo and now it's `32.6 kb`.

Reply:
Nice!